app/models/models.js

```diff
@@ -11,11 +11,37 @@ const fetchTopicsData = () => {
       return result.rows;
     })
     .catch((err) => {
-      console.log(err + " in fetchTreasures");
+      console.log(err + " in fetchTopicsData");
     });
 };
 
+const fetchArticlesData = () => {
```

Comment:
I would probably try to keep the function names a bit more semantic - fetchAllArticles would be a good one!